<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataTableWig</class>
 <widget class="QWidget" name="dataTableWig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>秒</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>毫秒</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>微秒</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>纳秒</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X轴结束位置：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QRadioButton" name="radioButton_Oneglobar">
          <property name="text">
           <string>单窗口定位</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QRadioButton" name="radioButton_globar">
          <property name="text">
           <string>全窗口定位</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QPushButton" name="btnClearTracer">
          <property name="text">
           <string>清空定位</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X轴起始位置：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="btn_locatorData">
          <property name="text">
           <string>定位数据</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>单位：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_start"/>
        </item>
        <item row="4" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_end"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
